src/influencetx/finances/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from influencetx.legislators.models import Legislator
+from influencetx.core import constants
+
+
+class FinancialDisclosure(models.Model):
+    year = models.CharField(max_length=4)
+    elected_officer = models.CharField(max_length=100, blank=True)
+    candidate = models.CharField(max_length=100, blank=True)
+    legislator = models.ForeignKey(Legislator,
+                                   on_delete=models.CASCADE,
+                                   related_name='financial_disclosures')
+
+    def __str__(self):
+        return self.year + " - " + self.elected_officer
+
+
+class Stock(models.Model):
+    name = models.CharField(max_length=100)
+    held_by = models.CharField(max_length=50,
+                               choices=constants.HELD_BY_CHOICES)
+    num_shares = models.CharField(max_length=100)
```

Comment:
This was the part I debated about most, it is actually always one of like 5 options `100 TO 499`, `500 TO 999`, etc.
It could be an enum, or some fancy way of storing a range. And some of the other information we may import later from the financial disclosures share these options, and some almost share them. I wasn't sure of the best way to store the data so I went with the simplest.